src/components/SceneGraph.js

```diff
@@ -176,8 +178,15 @@ export default class SceneGraph extends React.Component {
         let className = 'option' + (option.value === this.state.value ? ' active' : '');
         return (
           <div key={idx} className={className} value={option.value}
-            dangerouslySetInnerHTML={{__html: option.html}}
-            onClick={() => {this.handleEntityClick(option.value)}}/>
+            onClick={this.setValue.bind(this, option.value)}>
+            <span dangerouslySetInnerHTML={{__html: option.html}}></span>
+              <span className="icons">
+                <a href="#" onClick={() => cloneEntity(option.value)}
+                  title="Clone entity" className="button fa fa-clone"></a>
+                <a href="#" onClick={(e) => {e.stopPropagation(); removeEntity(option.value)}}
```

Comment:
don't need parens around `e`. can also use `evt` or `event` to be clearer

remove `href`s